.pipelines/cni/k8s-e2e/k8s-e2e-step-template.yaml

```diff
@@ -0,0 +1,57 @@
+parameters:
+  testName: ""
+  name: ""
+  clusterName: ""
+  ginkgoFocus: ""
+  ginkgoSkip: ""
+  os: ""
+  processes: "" # Number of parallel processes
+  attempts: ""
+
+
+steps:
+  - script: |
+      set -ex
+
+      # ginkgoSkip cant handle only |LinuxOnly. Need to have check
+      if ${{ lower(and(ge(length(parameters.ginkgoSkip), 1), eq(parameters.os, 'windows'))) }}
+      then
+        SKIP="|LinuxOnly"
+      elif ${{ lower(eq(parameters.os, 'windows')) }}
+      then
+        SKIP="LinuxOnly"
+      fi
+
+      # Taint Linux nodes so that windows tests do not run on them
+      if ${{ lower(eq(parameters.os, 'windows')) }}
+      then
+        kubectl taint node $(kubectl get nodes -owide \
```

Comment:
this can be replaced with  `kubectl taint nodes node-role.kubernetes.io/control-plane:NoSchedule -l kubernetes.io/os=linux`
Since we want to taint linux nodes, and not just Ubuntu ones.

Reply:
Would `-l kubernetes.azure.com/mode=system` would be better? We want to run the tests on anything that is not a system node.

Reply:
Yeah sure